### PR TITLE
Update and Optimize Circle Packer 2.0

### PR DESCRIPTION
**Pull request by azaleacolburn**

### CODE REVIEW

**Comment by rutmanz** [comment]
What's going on with the commit history? There's stuff from a month ago here

**Comment by azaleacolburn** [comment]
> What's going on with the commit history? There's stuff from a month ago here

Um, there's stuff I never merged that has nothing to do with the PR, but was just on my fork. It's really small stuff, mostly just cleaning stuff up, and making it look pretty

Nvm, there might be some stuff I really don't want here. It just makes the PR look sloppy. Only the later stuff will have an effect.

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version = 1
+
+[[analyzers]]
+name = "javascript"
+enabled = true
```
Comment (rutmanz) [inline .deepsource.toml:5]: Did you mean to commit this? It looks like a local config file

File: src/frontend/grid/index.ts [hunk 2]
```diff
@@ -102,8 +102,11 @@ export async function run(memberlist:CluckMember[]) {
 	addEventListener("resize", redrawRows);
 })();
 
+export let timeLoggedIn;
+
 async function refreshLoggedIn() {
 	let membersIn:LoggedIn;
+	timeLoggedIn = Date.now();
```
Comment (rutmanz) [inline src/frontend/grid/index.ts:109]: It looks like this is never used in any file

**Review by rutmanz — CHANGES_REQUESTED** [review]
I'm not really sure what this PR changes. Did you forget to push something?

File: src/frontend/grid/index.ts (hunk 2 above)
Comment (azaleacolburn) [reply src/frontend/grid/index.ts:109]: oh ya, that was from when I started to re-write the circle-sizing system but never got around to it

**Pull request closed without merging**